packages/core/orchestration/src/transaction/types.ts

```diff
@@ -237,6 +242,7 @@ export type TransactionFlow = {
   transactionId: string
   metadata?: {
     eventGroupId?: string
+    parentIdempotencyKey?: string
```

Comment:
suggestion: should we store the parent workflow id, I am thinking for example for the UI and for the telemetry that way we don't have to infer it from the idempotency key which in theory can be provided by the consumer. does that make sense?

Reply:
I will let this for another iteration where we can potentially have it as an array.
I believe when we focus in the UI we can tackle this and decide which approach is better.